client/package.json

```diff
@@ -66,6 +68,7 @@
     "eslint-config-prettier": "^8.1.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-react": "^7.22.0",
+    "nodemon": "^2.0.7",
```

Comment:
This is not required, and can be removed.

Reply:
@purvankbhiwgade  you know why we dont need it right?